### PR TITLE
refactor: user views

### PR DESCRIPTION
**Pull request by theus77**
|Q              |A  |
|---------------|---|
|Bug fix?       |Y|
|New feature?   |Y|	
|BC breaks?     |N|	
|Deprecations?  |N|	
|Fixed tickets  |N|	

#Fixes:

- Order on date column
- comma list without ugly space in entyti table

#Refactor

- Hide loop.index in non sortable data table view

#Feature:

- Header with a icon

### CODE REVIEW

**Review by Davidmattei — APPROVED** [review]

File: src/Controller/UserController.php [hunk 1]
```diff
@@ -57,17 +57,16 @@ public function __construct(LoggerInterface $logger, FormRegistryInterface $form
     public function indexAction(Request $request)
     {
         $table = new EntityTable($this->userService);
-
-        $table->addColumn('user.index.column.id', 'id');
         $table->addColumn('user.index.column.username', 'username');
         $table->addColumn('user.index.column.displayname', 'name');
-        $table->addColumn('user.index.column.email_notification', 'emailNotification', [true => 'fa fa-check-square-o', false => 'fa fa-square-o']);
+        $column = $table->addColumn('user.index.column.email_notification', 'emailNotification', [true => 'fa fa-check-square-o', false => 'fa fa-square-o']);
+        $column->setIconClass('fa fa-bell');
```
Comment (dameert) [inline src/Controller/UserController.php:63]: I see $colum, then $createdColumn, and that one is used multiple times for different colums.

As we are defining stuff here, why not daisy chain the commands? Just as form configuration does?

```suggestion
        $table
            ->addColumn('user.index.column.email_notification', 'emailNotification', [true => 'fa fa-check-square-o', false => 'fa fa-square-o'])
            ->setIconClass('fa fa-bell');
```

File: src/Form/Data/TableColumn.php [hunk 2]
```diff
@@ -16,7 +16,8 @@ final class TableColumn
     private ?string $iconProperty = null;
     private ?bool $dateTimeProperty = null;
     private ?bool $dataLinks = null;
-    private ?bool $arrayOfStrings = null;
+    private string $class = 'nowrap';
```
Comment (dameert) [inline src/Form/Data/TableColumn.php:19]: Hard dependency on the styling system that is used.. Maybe use constants and translate them inside the main template of twig?
```suggestion
    private string $class = 'nowrap';
```

File: src/Controller/UserController.php (hunk 1 above)
Comment (theus77) [reply src/Controller/UserController.php:63]: Agree see my coming PR

File: src/Form/Data/TableColumn.php (hunk 2 above)
Comment (theus77) [reply src/Form/Data/TableColumn.php:19]: Agree see my coming PR

**Pull request merged**